Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/rest/ConceptLookup.java

```diff
@@ -81,4 +102,123 @@
 
         return new ResponseEntity<String>(returnMsg.getXML(xmlEntries), HttpStatus.OK);
     }
+
+    @RequestMapping(value = "rest/ConceptLookupRdf/{word}/{pos}", method = RequestMethod.GET, produces = "application/xml")
```

Comment:
This should be available at the same endpoint as the regular xml (rest/ConceptLookup/{word}/{pos}). If you set produces to "application/rdf+xml", Spring will call this method when some goes to .ConceptLookup/word/pos.rdf.